cmd/pctl/add.go

```diff
@@ -204,7 +204,7 @@ func addProfile(c *cli.Context) error {
 	}
 	installer := install.NewInstaller(install.Config{
 		GitClient:        g,
-		RootDir:          filepath.Join(dir, profileName),
+		RootDir:          filepath.Join(dir, subName),
```

Comment:
with this change wdyt about making `--name` a mandatory flag? Atm its optional and defaults to `pctl-profile` (:vomiting_face: )

Reply:
actually I was thinking about changing this, good shout, I'll add changes

Reply:
sweet!